lib/rounded_loading_button.dart

```diff
@@ -84,6 +91,8 @@ class RoundedLoadingButton extends StatefulWidget {
       this.curve = Curves.easeInOutCirc,
       this.errorColor = Colors.red,
       this.successColor,
+      this.resetDuration = const Duration(seconds: 15),
```

Comment:
Only concern with this is we'd be introducing this behaviour as the default. Maybe best to have no auto reset in case people aren't aware of the behaviour.

Reply:
Hey, @chrisedg87 thanks for the review, People should be aware of the default behaviour as it is mentioned in the docs. The idea of auto-reset animation is to not block the button if something unexpected happens so if people want to use auto-reset animation they might also want to define the duration.

Reply:
Ok I'm convinced. Although we will need to make a note to update the docs.